v2/jdbc-to-googlecloud/src/main/java/com/google/cloud/teleport/v2/options/JdbcToBigQueryOptions.java

```diff
@@ -195,28 +209,30 @@ public interface JdbcToBigQueryOptions
 
   void setNumPartitions(Integer numPartitions);
 
-  @TemplateParameter.Long(
-      order = 16,
+  @TemplateParameter.Text(
+      order = 17,
       optional = true,
       description = "Lower bound of partition column.",
       helpText =
-          "The lower bound to use in the partition scheme. If not provided, this value is automatically inferred by Apache Beam for the supported types.")
-  Long getLowerBound();
+          "The lower bound to use in the partition scheme. If not provided, this value is automatically inferred by Apache Beam for the supported types. `datetime` "
+              + "partitionColumnType accepts lower bound in the format `yyyy-MM-dd HH:mm:ss.SSSZ`. For example, `2024-02-20 07:55:45.000+03:30`.")
+  String getLowerBound();
 
-  void setLowerBound(Long lowerBound);
+  void setLowerBound(String lowerBound);
```

Comment:
though we indeed changed the parameter type here. Should have no breaking change as parameters are passed in as command line options